### PR TITLE
testing/singularity: new aport

### PR DESCRIPTION
**Pull request by OTLabs**
https://www.sylabs.io/singularity/
Application containers focused on reproducibility for scientific computing and HPS world.

### CODE REVIEW

File: testing/singularity/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+# Maintainer: Oleg Titov <oleg.titov@gmail.com>
+pkgname=singularity
+pkgver=3.1.1
+pkgrel=0
+pkgdesc="Application containers focused on reproducibility for scientific computing and HPC world."
+url="https://www.sylabs.io/singularity/"
+arch="all"
+license="BSD-3-Clause BSD-3-Clause-LBNL"
+options="suid"
+#depends="squashfs-tools"
```
Comment (Ikke) [inline testing/singularity/APKBUILD]: Why is this commented out?

Comment (OTLabs) [reply testing/singularity/APKBUILD]: @Ikke Installation instruction (https://github.com/sylabs/singularity/blob/master/INSTALL.md) suggest to install several packages on Ubuntu. I had my doubts about some of them, in particular about `libgpgme11-dev` `pkg-config` `squashfs-tools`, so I was moving them from `makedepends` to `#depends` to check the build.

**Comment by tcely** [comment]
This is resurrecting the same software from https://github.com/alpinelinux/aports/commit/22fe867f12c9b270a510b564be2e008c83229420 and https://github.com/alpinelinux/aports/commit/2e168db253a5cf66d0fb58b40469121207347b06 correct?

**Comment by OTLabs** [comment]
@tcely Yes, this is the same software. I have missed it in `unmaintained/`. Should I use it as a base and move to `testing/`?

**Comment by tcely** [comment]
@OTLabs You might want to look over the existing files. If you find something useful there great. Either way your branch should probably remove the `singularity` package from `unmaintained` to avoid the confusion of having two.

File: testing/singularity/APKBUILD (hunk 1 above)
Comment (OTLabs) [reply testing/singularity/APKBUILD]: @Ikke I have enabled `depends="squashfs-tools"` as `squashfs-tools` are used during `singularity build` stage.

**Comment by OTLabs** [comment]
@tcely I have removed `unmaintained/singularity` in second commit. The build system was completely changed, now it is golang based.

**Comment by tcely** [comment]
@OTLabs the commit message could be improved.

```
unmaintained/singularity: removed

absorbed by new package with new build system
```

**Comment by OTLabs** [comment]
@tcely Improved! Thank you for suggestion!

**Pull request closed without merging**